lib/rules/no-deprecated-api.js

```diff
@@ -760,17 +763,21 @@ module.exports = {
          * @returns {void}
          */
         function reportItem(node, name, info) {
+            const messageId = info.removed ? "removed" : "deprecated"
+            const data = {
+                name,
+                version: info.since,
+                replace: toReplaceMessage(info.replacedBy, version),
+            }
+            if (info.removed) Object.assign(data, { removed: info.removed })
```

Comment:
Did you do this to prevent the tests from failing?
```suggestion
            const data = {
                name,
                version: info.since,
                removed: info.removed,
                replace: toReplaceMessage(info.replacedBy, version),
            }
```

Reply:
Now we are getting a type error.: https://github.com/eslint-community/eslint-plugin-n/actions/runs/8719739313/job/23919733364?pr=240#step:6:5